### ISSUE TITLE
conda search doesn't raise any errors

### ISSUE THREAD

**Issue by tyler-thetyrant** [conda/conda#5390]
With the most recent conda version (4.3.18)

```
> conda search tensorflow
Fetching package metadata ...........
```
That is the end of the output.

I know that `conda search -c conda-forge tensorflow` will return relevant results, but I believe some sort of explanation should be added if the said package is not available in the defaults channel.

**Comment by tyler-thetyrant** [comment]
Turns out,  search execute function does not raise `NoPackagesFoundError` at all.

**Comment by tyler-thetyrant** [comment]
This is fixed in conda 4.3.x. This will be merged up to 4.4.x soon.
